_layouts/default.html

```diff
@@ -1,8 +1,10 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="utf-8">
-    <title>{{site.data.tenant.name}} Documentation</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <title>{{site.data.tenant.name}} iPaaS Documentation</title>
```

Comment:
any particular reasons to have `iPaaS` here? How it will look for existing installations after this merge?

Reply:
it would write like `elastic.io` iPaaS Documentation or `Magicsoftware` iPaaS Documentation. Do you think this is problematic?

Reply:
Just removed the word `iPaaS` as we discussed offline.